sql/table.h

```diff
@@ -3054,10 +3054,10 @@ inline bool is_user_table(TABLE * table)
 bool is_simple_order(ORDER *order);
 
 void repoint_field_to_record(TABLE *table, uchar *old_rec, uchar *new_rec);
-bool update_generated_write_fields(const MY_BITMAP *bitmap, TABLE *table);
-bool update_generated_read_fields(uchar *buf, TABLE *table,
-                                  uint active_index= MAX_KEY);
-
+bool  update_generated_write_fields(const MY_BITMAP *bitmap, TABLE *table);
```

Comment:
Whitespace-only changes above the added declaration

Reply:
The same, fixed.